community_supplied/VendorSpecific/Cisco/IOS_XR/check-ios-xr-processes-cpu-netconf.rule

```diff
@@ -0,0 +1,129 @@
+/*
+ * Detects cisco IOS-XR processes(daemons) CPU utilization threshold breaches
+ * and notifies when anomalies are found.
+ */
+healthbot {
+    topic system.processes {
+        rule check-ios-xr-processes-cpu-netconf {
+            keys [ pid process-name ];
+            synopsis "System processes cpu analyzer";
+            description "Collects system process's cpu utilization periodically and notifies anomalies when daemon CPU usage exceed threshold";
+            sensor proc {
+                iAgent {
+                    file XRCpuProcessesTable.yml;
+                    table XRCpuProcessesTable;
+                    frequency 60s;
```

Comment:
Can you please make this an offset?

Reply:
Offset is only for trigger frequency and time-range and not for sensor in rule. For sensor, we need to configure under ingest settings.